specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/examples/ListOperations.json

```diff
@@ -1187,42 +1187,6 @@
                         "Origin": "user"
                     },
                     {
-                        "Name": "Microsoft.RecoveryServices/Vaults/providers/Microsoft.Insights/logDefinitions/read",
```

Comment:
why did you decide to remove this part? It's good to include more and cover more fields, we need to make sure the wire matches what's in the swagger, that's the purpose of having the example covering more.